src/main/java/control/TimelineController.java

```diff
@@ -0,0 +1,102 @@
+package control;
+
+import data.Camera;
+import data.CameraShot;
+import data.CameraTimeline;
+import data.CameraType;
+import data.ScriptingProject;
+import gui.CameraShotBlock;
+import gui.ShotblockUpdatedEvent;
+import gui.TimelinesGridPane;
+import gui.RootPane;
+import gui.TimetableBlock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class that controls the timeline.
+ * @author alex
+ */
+public class TimelineController {
+
+    private TimelinesGridPane timelinePane;
+    private RootPane rootPane;
+
+    private List<CameraTimeline> cameraTimelines;
```

Comment:
cameraTimeLines list thing is part of the project, why do you create a separate list for that?